app/src/app/components/Routes/Router.tsx

```diff
@@ -0,0 +1,22 @@
+import {Route, Routes} from "react-router-dom";
```

Comment:
Tiny point, but the folder name `Routes` should probably be lower-cased like the others